syft/src/main/java/org/openmined/syft/execution/SyftJob.kt

```diff
@@ -71,14 +71,15 @@ class SyftJob internal constructor(
                 worker,
                 config,
                 JobRepository(
-                    JobLocalDataSource(),
-                    JobRemoteDataSource(config.getDownloader())
+                    JobId(modelName, version),
```

Comment:
Better to create a static factory method to instantiate all these job local/remote data sources from the config. Use that method everywhere